behave_html_pretty_formatter/html_pretty.py

```diff
@@ -221,6 +221,20 @@ def generate_feature(self, formatter):
                     with a(onclick="expander('expand_all_failed')", href="#"):
                         div("[Expand All Failed]", cls="feature-summary-row button")
 
+            if formatter.additional_info:
```

Comment:
Toggle summary will not work for this, if it is not inside `feature-summary-container`. Either add `feature-aditional-info-container` to javascript. Or better, put this block inside `feature-summary-container` (one more indent + additional change to CSS may be needed).